lib/fluent/plugin/in_tail.rb

```diff
@@ -775,6 +936,11 @@ def initialize(target_info, pe, log, read_from_head, follow_inodes, update_watch
       attr_reader :line_buffer_timer_flusher
       attr_accessor :unwatched  # This is used for removing position entry from PositionFile
       attr_reader :watchers
+      attr_accessor :group_watcher
+
+      def group_watcher=(group_watcher)
```

Comment:
`attr_accessor :group_watcher` creates `group_watcher=` method.
What is the expected behavior for this method?

Reply:
Thanks for pointing this out! I have updated the code.